themes/cucumber-sb/static/js/site.js

```diff
@@ -128,27 +136,17 @@ var supportedLanguages = [
   "scala"
 ]
 
-var defaulLanguage = 'java'
+var defaultLanguage = 'java'
 
 ready(function() {
   var selectedLang = getLangFromUrl();
   if((selectedLang == '' || selectedLang == null)){
-    if(localStorage.getItem('language') == ''){
-      showOnly(defaulLanguage)
-      localStorage.setItem('language', defaulLanguage)
-    }else{
-      showOnly(localStorage.getItem('language'))
-    }
+    setDefaultLang()
```

Comment:
would `showDefaultLang` be a better name maybe?

Reply:
@mattwynne ,
I think `setDefaultLang` is better as we don't try to show the default language by this function; rather set it. `showOnly` function shows the default language.